### ISSUE TITLE
L R reverse

### ISSUE THREAD

**Issue by gtk2k** [resonance-audio/resonance-audio-web-sdk#29]
When I create and run "treasure-hunt.n.html" that modified the Tresure-Hunt sample to load the js file of src (direct_src) directly, L and R will sound reversely Yes. why?
[repository:forked](https://github.com/gtk2k/resonance-audio-web-sdk/tree/master/examples)

**Comment by Ghorthalon** [comment]
Yes, I can confirm that when setting the AmbisonicOrder to a higher value than 1, left and right seem to swap. I suppose this is not intended behavior?

**Comment by capnmidnight** [comment]
From what I can tell, this appears to have happened because the Omnitone code base was ported from a version originally written for a C++-based game engine, wherein the coordinate system is left-handed. Resonance/Omnitone compute audio in the right-handed coordinate system, so for that game engine the rotations had to be flipped to match the World. But WebAudio is right-handed, so that conversion isn't necessary.

Since this repo hasn't been updated in several years, I forked the two projects and fixed the issue here: https://github.com/capnmidnight/resonance-audio-web-sdk

**Comment by Code6226** [comment]
@capnmidnight Looks like that link is 404ing now. What change did you make to fix it?
